server/rest/tale.py

```diff
@@ -320,3 +322,31 @@ def stream():
             yield zip.footer()
 
         return stream
+
+    @access.public
+    @autoDescribeRoute(
+        Description('Generate the Tale manifest')
+        .modelParam('id', model='tale', plugin='wholetale', level=AccessType.READ)
+        .param(name='license',
+               required=True,
+               description='The SPDX of the license that the Tale is under.'
+                           'For example, `CC0-1.0`')
+        .jsonParam(name='itemIds',
+                   required=False,
+                   description='A list of item ids of files that are aggregated.\n'
+                               'Example: ["item1", "item2", "item3"]')
+        .errorResponse('ID was invalid.')
+    )
+    def generateManifest(self, tale, license, itemIds=list()):
+        """
+        Creates a manifest objbect and returns the contents.
+        :param tale: The Tale whose information is being used
+        :param license: The license that the Tale is under
+        :param itemIds: An optional list of items to include in the manifest
+        :return: A JSON structure representing the Tale
+        """
+
+        manifest_doc = Manifest(license, itemIds)
+        user = self.getCurrentUser()
+        manifest_doc.generate_manifest(user, tale)
```

Comment:
I feel like this belongs to `Manifest.__init__`. Is there any scenario where `Manifest()` wouldn't be followed by `generate_manifest`?

Reply:
There wasn't anywhere else in the code where we were using a standalone Manifest object that wasn't filled out, so I moved it to init in 651f7a4